lib/kanta/translations/singular_translation/singular_translation_spec.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Kanta.Translation.SingularTranslationSpec do
+  alias Kanta.Translations.{Locale, Message, SingularTranslation}
+  alias Kanta.Types
+
+  @type t() :: %SingularTranslation{
+    id: Types.field(Types.id()),
+    original_text: Types.field(String.t()),
+    translated_text: Types.field(String.t()),
+    locale: Types.field(Locale.t()),
+    locale_id: Types.field(Types.id()),
```

Comment:
Maybe `Locale.id()` instead?

Reply:
There is no such type as `Kanta.Translations.Locale.id/0`.